packages/@o3r/third-party/package.json

```diff
@@ -20,6 +20,10 @@
     "tslib": "^2.5.3",
     "uuid": "^9.0.0"
   },
+  "peerDependencies": {
+    "@angular-devkit/schematics": "~16.0.5",
```

Comment:
these 2 can pass as optional peer deps as they are used only in schematics/generators ?